sale_last_price_info/models/product.py

```diff
@@ -0,0 +1,29 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    @api.one
+    def _get_last_sale(self):
+        """ Get last sale price, last sale date and last customer """
+        lines = self.env['sale.order.line'].search(
+            [('product_id', '=', self.id),
+             ('state', 'in', ['confirmed', 'done'])]).sorted(
+            key=lambda l: l.order_id.date_order, reverse=True)
+        self.last_sale_date = lines[:1].order_id.date_order
+        self.last_sale_price = lines[:1].price_unit
+        self.last_customer = lines[:1].order_id.partner_id
+
+    last_sale_price = fields.Float(
+        string='Last Sale Price', compute='_get_last_sale')
+    last_sale_date = fields.Date(
+        string='Last Sale Date', compute='_get_last_sale')
+    last_customer = fields.Many2one(
```

Comment:
Pedro, please read again.
You don't recognize this user, so, but the PR has still been accepted.

> It should already be the case but every now and then I happen to see some misleading fields/variable names.

> One2Many and Many2Many fields should always have _ids as suffix (example: sale_order_line_ids)
Many2One fields should have _id as suffix (partner_id, user_id, ...)

Isn't this clear enough?

> Same for the variables name. Especially since the new api. If we have a record or recordset, don't name the variable with _id, when I see a _id in a variable name, I expect a number, not a record

**variables name**.
And we do the same.
Example:

```pyhon
partners = record.partner_ids
```